src/Sprite.ts

```diff
@@ -450,37 +544,38 @@ export class Sprite extends SpriteBase {
     this.clones.push(clone);
 
     // Trigger CLONE_START:
-    const triggers = clone.triggers.filter(tr =>
+    const triggers = clone.triggers.filter((tr) =>
       tr.matches(Trigger.CLONE_START, {}, clone)
     );
-    this._project._startTriggers(
-      triggers.map(trigger => ({ trigger, target: clone }))
+    void this._project._startTriggers(
+      triggers.map((trigger) => ({ trigger, target: clone }))
     );
   }
 
-  deleteThisClone() {
+  public deleteThisClone(): void {
     if (this.parent === null) return;
 
-    this.parent.clones = this.parent.clones.filter(clone => clone !== this);
+    this.parent.clones = this.parent.clones.filter((clone) => clone !== this);
 
     this._project.runningTriggers = this._project.runningTriggers.filter(
       ({ target }) => target !== this
     );
   }
 
-  andClones() {
-    return [this, ...this.clones.flatMap(clone => clone.andClones())];
+  // TODO: is this necessary now that the clone hierarchy seems to be flattened?
+  public andClones(): Sprite[] {
+    return [this, ...this.clones.flatMap((clone) => clone.andClones())];
```

Comment:
I don't think this is necessary, and I've flattened many arrays in my recent years 📦

Reply:
Actually, it looks like `this.clones` only contains direct clones. A clone of a clone of a clone(...) is not included in it.